build_runner/test/generate/build_test.dart

```diff
@@ -473,94 +475,38 @@ void main() {
 
   group('incremental builds with cached graph', () {
     test('one new asset, one modified asset, one unchanged asset', () async {
-      var graph = await AssetGraph.build([], new Set(), 'a', null);
-      var bId = makeAssetId('a|lib/b.txt');
-      var bCopyNode = new GeneratedAssetNode(
-          0, bId, false, true, makeAssetId('a|lib/b.txt.copy'),
-          lastKnownDigest: computeDigest('b'));
-      graph.add(bCopyNode);
-      var bNode =
-          makeAssetNode('a|lib/b.txt', [bCopyNode.id], computeDigest('b'));
-      graph.add(bNode);
+      var buildActions = [copyABuildAction];
 
+      // Initial build.
       var writer = new InMemoryRunnerAssetWriter();
-      await writer.writeAsString(makeAssetId('a|lib/b.txt'), 'b');
-      await testActions([
-        copyABuildAction
-      ], {
-        'a|web/a.txt': 'a',
-        'a|lib/b.txt.copy': 'b',
-        'a|lib/c.txt': 'c',
-        'a|$assetGraphPath': JSON.encode(graph.serialize()),
-      }, outputs: {
-        'a|web/a.txt.copy': 'a',
-        'a|lib/c.txt.copy': 'c',
-      }, writer: writer);
-    });
-
-    test('invalidates generated assets based on graph age', () async {
```

Comment:
note that I deleted this test, it wasn't applicable any more and should have been deleted during the migration to content hashes